manifests/s/Stretchly/Stretchly/1.11.0/Stretchly.Stretchly.installer.yaml

```
@@ -0,0 +1,25 @@
+PackageIdentifier: Stretchly.Stretchly
+PackageVersion: 1.11.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+InstallModes:
+- interactive
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/hovancik/stretchly/releases/download/v1.11.0/Stretchly-Setup-1.11.0.exe
+  InstallerSha256: c2592e60e28aed1c66736b370bd4a5f934503c24818ad5a1015b0a9251f7542c
+  InstallerSwitches:
+    Custom: /CURRENTUSER
+- Architecture: x86
```

```suggestion
- Architecture: x64
```
This application only runs on 64-bit version of Windows.